### PR TITLE
relationship between post and user set

### PR DESCRIPTION
**Pull request by holyspecter**

### CODE REVIEW

File: src/AppBundle/DataFixtures/ORM/LoadFixtures.php [hunk 1]
```diff
@@ -88,6 +88,10 @@ private function loadPosts(ObjectManager $manager)
         $manager->persist($user);
         $manager->flush();
 
+        $author = $manager->getRepository('AppBundle:User')->findOneBy([
```
Comment (panayotovyura) [inline src/AppBundle/DataFixtures/ORM/LoadFixtures.php]: you can use findOneByEmail

File: src/AppBundle/Entity/Post.php [hunk 2]
```diff
@@ -37,6 +37,8 @@ class Post
     const VOTING_MOST_POPULAR = 1;
```
Comment (panayotovyura) [inline src/AppBundle/Entity/Post.php]: you can use chain pattern here

File: src/AppBundle/Controller/Admin/BlogController.php [hunk 3]
```diff
@@ -52,9 +52,14 @@ class BlogController extends Controller
      */
     public function indexAction()
     {
+        // todo: we need to have user from database in session
+        $user = $this->getDoctrine()
+            ->getRepository('AppBundle:User')
+            ->findOneByEmail($this->getUser()->getEmail());
+
         $em = $this->getDoctrine()->getManager();
         $posts = $em->getRepository('AppBundle:Post')->findBy([
```
Comment (Ishugaev) [inline src/AppBundle/Controller/Admin/BlogController.php]: We need to add logic according to user role. Admin should be able seeing all post except Drafts. User should be able seeing only own posts

File: src/AppBundle/DataFixtures/ORM/LoadFixtures.php (hunk 1 above)
Comment (holyspecter) [inline src/AppBundle/DataFixtures/ORM/LoadFixtures.php]: the less magic - the better, imo

File: src/AppBundle/Entity/Post.php (hunk 2 above)
Comment (holyspecter) [inline src/AppBundle/Entity/Post.php]: o_0

File: src/AppBundle/DataFixtures/ORM/LoadFixtures.php (hunk 1 above)
Comment (panayotovyura) [inline src/AppBundle/DataFixtures/ORM/LoadFixtures.php]: what about __toString() ;)

Comment (holyspecter) [inline src/AppBundle/DataFixtures/ORM/LoadFixtures.php]: that's php built-in magic, therefore it's ok :)

File: src/AppBundle/Entity/Post.php (hunk 2 above)
Comment (holyspecter) [inline src/AppBundle/Entity/Post.php]: @panayotovyura done

File: src/AppBundle/Controller/Admin/BlogController.php [hunk 4]
```diff
@@ -84,6 +87,12 @@ public function newAction(Request $request)
         // However, we explicitly add it to improve code readability.
         // See http://symfony.com/doc/current/best_practices/forms.html#handling-form-submits
         if ($form->isSubmitted() && $form->isValid()) {
+            // todo: we need to have user from database in session
+            $user = $this->getDoctrine()
+                ->getRepository('AppBundle:User')
+                ->findOneByEmail($this->getUser()->getEmail());
+
+            $post->setAuthor($user);
             $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
```
Comment (panayotovyura) [inline src/AppBundle/Controller/Admin/BlogController.php]: now you can use chain here ;)

**Comment by panayotovyura** [comment]
approved

**Comment by alexandr-kalenyuk** [comment]
What's status of this PR?

**Pull request merged**